tools/jest.bzl

```diff
@@ -0,0 +1,21 @@
+"From: https://github.com/bazelbuild/rules_nodejs/blob/master/examples/jest/jest.bzl"
+
+load("@npm//jest-cli:index.bzl", _jest_test = "jest_test")
+
+def jest_test(name, srcs, deps, jest_config = "//:jest.config.js", **kwargs):
+    "A macro around the autogenerated jest_test rule"
+    templated_args = [
+        "--no-cache",
+        "--no-watchman",
```

Comment:
How do I run Jest watch mode with this change?

Reply:
`ibazel` would replace that I believe